Crane/app/src/test/java/androidx/compose/samples/crane/data/DestinationsRepositoryTest.kt

```diff
@@ -0,0 +1,45 @@
+package androidx.compose.samples.crane.data
+
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldNotBe
+import org.junit.Test
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+
+internal class DestinationsRepositoryTest {
+
+    val madrid = ExploreModel(
```

Comment:
IMO it's not our concerns to fill strings like real, I prefer to make a test factory to create types and create random string  by using something like UUID.toString() inside it.